contracts/src/arbitration/IArbitrator.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8;
+
+import "./IArbitrable.sol";
+
+/**
+ * @title Arbitrator
+ * Arbitrator interface for CourtV2.
+ * This interface for the most part follows the ERC-792 standard but also allows the appeal crowdfunding on arbitrator's side.
+ * When developing arbitrator contracts we need to:
+ * - Define the functions for dispute creation (createDispute). Don't forget to store the arbitrated contract and the disputeID (which should be unique, may nbDisputes).
+ * - Define the function for appeal crowdfunding (fundAppeal) in order to appeal the ruling.
+ * - Define the functions for cost display (arbitrationCost and appealCost).
+ * - Allow giving rulings. For this a function must call arbitrable.rule(disputeID, ruling).
+ */
+interface IArbitrator {
+    enum DisputeStatus {
+        Waiting,
+        Appealable,
+        Solved
+    }
+
+    /**
+     * @dev To be emitted when a dispute is created.
+     * @param _disputeID ID of the dispute.
+     * @param _arbitrable The contract which created the dispute.
+     */
+    event DisputeCreation(uint256 indexed _disputeID, IArbitrable indexed _arbitrable);
+
+    /**
+     * @dev To be emitted when a dispute can be appealed.
+     * @param _disputeID ID of the dispute.
+     * @param _arbitrable The contract which created the dispute.
+     */
+    event AppealPossible(uint256 indexed _disputeID, IArbitrable indexed _arbitrable);
+
+    /**
+     * @dev To be emitted when the current ruling is appealed.
+     * @param _disputeID ID of the dispute.
+     * @param _arbitrable The contract which created the dispute.
+     */
+    event AppealDecision(uint256 indexed _disputeID, IArbitrable indexed _arbitrable);
+
+    /**
+     * @dev Create a dispute. Must be called by the arbitrable contract.
+     * Must pay at least arbitrationCost(_extraData).
+     * @param _choices Amount of choices the arbitrator can make in this dispute.
+     * @param _extraData Can be used to give additional info on the dispute to be created.
+     * @return disputeID ID of the dispute created.
+     */
+    function createDispute(uint256 _choices, bytes calldata _extraData) external payable returns (uint256 disputeID);
+
+    /**
+     * @dev Compute the cost of arbitration. It is recommended not to increase it often, as it can be highly time and gas consuming for the arbitrated contracts to cope with fee augmentation.
+     * @param _extraData Can be used to give additional info on the dispute to be created.
+     * @return cost Required cost of arbitration.
+     */
+    function arbitrationCost(bytes memory _extraData) external view returns (uint256 cost);
+
+    /**
```

Comment:
This is now an internal function of the Arbitrator which does not need standardization so it should be possible to remove it just like appeal() has been removed.